lua/telescope/utils.lua

```diff
@@ -259,13 +259,14 @@ utils.strdisplaywidth = (function()
 
     return function(str, col)
       local startcol = col or 0
+      str = tostring(str)
```

Comment:
To-stringing here to mimic the API of [`vim.fn.strdisplaywidth`](https://github.com/neovim/neovim/blob/48e80572855042ceb1d9191bc11f8b60275426eb/src/nvim/eval/funcs.c#L10007). Let me know if you'd like me to to-string the number (idx) at the callsite instead

Reply:
you would want to talk to @delphinus because we are moving it to plenary see https://github.com/nvim-lua/plenary.nvim/pull/96

Reply:
I agree to add it. I didn't know `strdisplaywidth()` can deal with values other than str ;)

I think https://github.com/nvim-lua/plenary.nvim/pull/96 can take more time to merge. It is good to add here in advance..